pkg/kube/misc.go

```diff
@@ -70,11 +71,10 @@ func GetServiceInterfaceTarget(targetType string, targetName string, deducePort
 		return nil, fmt.Errorf("VAN service interfaces for pods not yet implemented")
 	} else if targetType == "service" {
 		var svcName, svcNamespace string = GetServiceName(targetName, namespace)
-		var formattedName = fmt.Sprintf("%s.%s", svcName, svcNamespace)
 		target := types.ServiceInterfaceTarget{
-			Name:      formattedName,
-			Service:   formattedName,
-			Namespace: namespace,
+			Name:      svcName,
```

Comment:
Since we are not allowing target-namespace in conjunction with service targets, I would think we should just use the specified targetName as the Name and the Service here, not set the namespace and and not try to interpret the targetName in anyway.

Reply:
We actually need the target namespace here as it is used to deduce ports below.

Reply:
Deducing the ports is an optional feature. We can still do that without actually recording the namespace in the target structure. It is fine if this only works when the target is the unqualified name and the namespace is the site namespace.